### PR TITLE
Sourcetype and image pull policy

### PR DESCRIPTION
**Pull request by rockb1017**

### CODE REVIEW

**Comment by luckyj5** [comment]
We should update the docs for the same? https://github.com/splunk/splunk-connect-for-kubernetes#managing-sck-log-ingestion-by-using-annotations. 

Also, can this be a breaking change for existing panels/ dashboards?

**Comment by luckyj5** [comment]
Thanks for updating the docs @rockb1017. 
What about breaking change? can this be a breaking change for existing panels/ dashboards?

Also, make sure we have a ticket to update the docs on Splunkbase as well (if needed). 
Thanks!

File: helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml [hunk 1]
```diff
@@ -46,6 +46,8 @@ k8sMetadata:
     - k8s-app
     - release
 
+sourcetypePrefix: "kube:"
```
Comment (luckyj5) [inline helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml:49]: Does it handle null values? (eg: "", ' ', <no value>, null)

Comment (rockb1017) [reply helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml:49]: good catch. it changes them to default kube:. let me fix it up.

Comment (luckyj5) [reply helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml:49]: Also, is ":" a standard separator? What if when prefix doesn't contain ":"?

Comment (rockb1017) [reply helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml:49]: it doesn't have negative impact if prefix doesn't end with ":"

Comment (luckyj5) [reply helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml:49]: Ok. May be change the default to  "kube" + ":" ?

Comment (rockb1017) [reply helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml]: sounds good. updated.

**Review by luckyj5 — APPROVED** [review]
Looks good. Thanks @rockb1017

**Pull request merged**